app/views/bookings/show.html.erb

```diff
@@ -125,6 +125,20 @@
                             <div>Total (A+B+C+D) = <%= @booking.cost %> </div>
                         </td>
                     </tr>
+                    <% elsif @booking.is_cancelled? %>
+                    <tr>
+                        <th>
+                            Cost Calculations
+                        </th>
+                        <td>
+                             <% if 0.5.day.from_now > DateTime.parse(@booking.start_date.to_s + "T" + @booking.time_in)%>
```

Comment:
not correct. this will be 0.5 day from when someone sees this booking page because thats when the code runs... instead 0.5 day is from date of cancellation...